### PR TITLE
Rewrite 02 and add CRAN

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: 02-graph_upstream.py [hunk 1]
```diff
@@ -1,100 +1,110 @@
...
-import os
-
-import github3
 import networkx as nx
 import requests
 from pkg_resources import parse_version
+import feedparser
 
+class VersionFromFeed:
+    ver_suffix_remove = ['release-', 'releases%2F', 'v']
```
Comment (CJ-Wright) [inline 02-graph_upstream.py]: I think these are prefixes.

Comment (isuruf) [reply 02-graph_upstream.py]: :smile:

**Comment by isuruf** [comment]
`Current number of out of date packages not PRed: 394`

**Comment by CJ-Wright** [comment]
That's not too bad. I was expecting it to be much more.

**Comment by justcalamari** [comment]
This all looks good to me. Happy this gets rid of all of the API calls in 02!

**Comment by CJ-Wright** [comment]
Thank you very much @isuruf!

**Pull request merged**